Scripts/Tank.gd

```diff
@@ -71,6 +74,8 @@ func _on_shoot_timeout():
 	_can_shoot = true
 
 func _on_killed() -> void:
+	GameManager.gameOver(playerName)
```

Comment:
To guarantee that the tank is destroyed before the `game over` logic is run, I would call this only at the end of the func.
What do you think?

Reply:
Yap, I think